zerver/lib/test_helpers.py

```diff
@@ -128,33 +132,39 @@ def cursor_executemany(self, sql, params=()):
 
 
 def find_key_by_email(address):
+    # type: (str) -> str
     from django.core.mail import outbox
     key_regex = re.compile("accounts/do_confirm/([a-f0-9]{40})>")
     for message in reversed(outbox):
         if address in message.to:
             return key_regex.search(message.body).groups()[0]
 
 def message_ids(result):
+    # type: (Dict[str, Any]) -> Set[str]
     return set(message['id'] for message in result['messages'])
```

Comment:
ah cool -- i'll take your word for it and do ints then

On Thu, Jun 2, 2016 at 10:53 PM, Tim Abbott <notifications@github.com>
wrote:

> In zerver/lib/test_helpers.py
> <https://github.com/zulip/zulip/pull/871#discussion_r65659456>:
>
> >      from django.core.mail import outbox
> >      key_regex = re.compile("accounts/do_confirm/([a-f0-9]{40})>")
> >      for message in reversed(outbox):
> >          if address in message.to:
> >              return key_regex.search(message.body).groups()[0]
> >
> >  def message_ids(result):
> > +    # type: (Dict[str, Any]) -> Set[str]
> >      return set(message['id'] for message in result['messages'])
>
> I think that's because the files that call this function (mostly
> zerver/tests/test_messages.py) are in the mypy exclude list, so there's
> nothing for it to check this annotation for consistency with :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zulip/zulip/pull/871/files/6c9a44881e949219dcd3f165f57751e7a1dcbc11#r65659456>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJxY_Z9kb_4LK6cNJPrfXaL29PWJAsxks5qH8FCgaJpZM4ItGOv>
> .
>